spec/std/int_spec.cr

```diff
@@ -13,6 +13,15 @@ private macro it_converts_to_s(num, str, **opts)
   end
 end
 
+private macro pending_win32_converts_to_s(num, str, **opts)
+  pending_win32 {{ "converts #{num} to #{str}" }} do
+    num = {{ num }}
+    str = {{ str }}
+    num.to_s({{ opts.double_splat }}).should eq(str)
+    String.build { |io| num.to_s(io, {{ opts.double_splat }}) }.should eq(str)
```

Comment:
Perhaps this could re-use `assert_prints` from `support/string`?

Reply:
I don't really think this is in the scope of this PR.
This method is just a copy of the method right above it, using pending_win32 instead of it.